tensorboard/webapp/webapp_data_source/BUILD

```diff
@@ -64,12 +64,14 @@ tf_ng_module(
         ":feature_flag_testing",
         ":http_client",
         ":http_client_testing",
+        "//tensorboard/webapp/angular:expect_angular_common_http",
         "//tensorboard/webapp/angular:expect_angular_core_testing",
         "//tensorboard/webapp/angular:expect_ngrx_store_testing",
         "//tensorboard/webapp/app_routing:app_root",
         "//tensorboard/webapp/core:types",
         "//tensorboard/webapp/feature_flag/store",
         "//tensorboard/webapp/feature_flag/store:types",
+        "@npm//@angular/common",
```

Comment:
Do you need this given that you have `expect_angular_common_http`?

Reply:
Huh; I was getting strict-deps errors, and adding this fixed it, but
I can’t seem to reproduce that now. Removed and test-synced; thanks.